config/application.rb

```diff
@@ -12,15 +12,14 @@
 
 module Feedback
   class Application < Rails::Application
-    require 'feedback'
+    config.load_defaults 5.1
```

Comment:
Does this mean we can get rid of some of the configs below?

Reply:
Based on the configs the `load_defaults` is setting, there aren't any duplicates that we think need removing: https://github.com/rails/rails/blob/v5.1.4/railties/lib/rails/application/configuration.rb#L57 

We'd normally kill off all of the default Rails comments in these too just to get rid of the noise. Is there a reason to keep them around?

Reply:
Cool, thanks for the explanation.

I'm very much in favour of deleting those comments & commented out code.

Reply:
👍  Let's deal with killing those comments as a separate PR.